EventFilter/CTPPSRawToDigi/src/CTPPSPixelDataFormatter.cc

```diff
@@ -188,6 +188,98 @@ void CTPPSPixelDataFormatter::interpretRawData(
   }
 }
 
+void CTPPSPixelDataFormatter::formatRawData(unsigned int lvl1_ID,
+                                            RawData& fedRawData,
+                                            const Digis& digis,
+                                            std::vector<PPSPixelIndex> iDdet2fed) {
+  std::map<int, vector<Word32> > words;
+  // translate digis into 32-bit raw words and store in map indexed by Fed
+  for (auto const& im : digis) {
+    m_allDetDigis++;
+    cms_uint32_t rawId = im.first;
+
+    m_hasDetDigis++;
+    const DetDigis& detDigis = im.second;
+    for (auto const& it : detDigis) {
+      m_DigiCounter++;
```

Comment:
This is never initialized in the constructor

Reply:
Done.